### ISSUE TITLE
Navigation - add inert attribute to content within a collapsed group

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11732]
**Is this a new component or an extension of an existing one?**
Navigation

**Describe the feature**
Related to the core fix for https://github.com/patternfly/patternfly/issues/7431, we should add the inert attribute to further prevent hidden content within a collapsed NavExpandable from being focused (specifically on the navSubnav element)

This should include a test (probably Cypress?) to check that content in an expanded NavExpandable can be focused, and cannot be focused when collapsed.

**Are there visuals for this feature? If applicable, please include examples for each state and for varying widths**
Include screenshots or links to Marvel or other mockups.

**Any other information?**

**Comment by Mash707** [comment]
@thatblindgeye are we talking about adding the inert attribute in the section part? make it `null` or `true` just like `hidden` based on `expandedState` 
https://github.com/patternfly/patternfly-react/blob/4ea82f7af5e42834f2a906ec5c7cc1912466ca2b/packages/react-core/src/components/Nav/NavExpandable.tsx#L144-L153

**Comment by thatblindgeye** [comment]
@Mash707 yep that's correct

**Comment by Mash707** [comment]
@kmcfaul working on this, could you assign this to me. Thank You
